modules/crypto/package.json

```diff
@@ -35,6 +35,10 @@
     "@babel/runtime": "^7.3.1",
     "@loaders.gl/loader-utils": "2.3.0-alpha.7",
     "@types/crypto-js": "*",
-    "crypto-js": "^3.0.0 || ^4.0.0"
+    "crypto-js": "^3.0.0 || ^4.0.0",
+    "sse4_crc32": "^6.0.1"
```

Comment:
This package isn't really used anywhere other than some commented out imports below.

Reply:
Yes, I tried to integrate this for additional acceleration but failed. Will make another attempt after this is published, and clean up later.